solr/core/src/test/org/apache/solr/cloud/SSLMigrationTest.java

```diff
@@ -49,7 +52,10 @@
 @AwaitsFix(bugUrl = "https://issues.apache.org/jira/browse/SOLR-12028") // 17-Mar-2018
 public class SSLMigrationTest extends AbstractFullDistribZkTestBase {
```

Comment:
unrelated change and `@Slow` annotation retained. accidental commit inclusion probably, it happens.

Reply:
yep, I spent a little bit of time trying to fix this test but didn't finish that